public/js/User.js

```diff
@@ -0,0 +1,13 @@
+/**/
+class User {
```

Comment:
You don't need this class because it only holds 'UI' class per user.
Instead, use one UI class and userName as string.